apps/skillrecordings/src/pages/epic-react/index.mdx

```diff
@@ -218,7 +240,27 @@ Epic React's release was the result of a ton of planning, care and attention bei
 
 **After 18 months of planning and building and an absolutely massive amount of effort, Epic React launched on September 29, 2020.**
 
-<Tweet tweetLink="kentcdodds/status/1310990076029972480" />
+{/* <Tweet tweetLink="kentcdodds/status/1310990076029972480" /> */}
+
+<Blockquote class="twitter-tweet">
```

Comment:
we can just hardcode them like this and twitter js widget takes care of the rest